packages/cashier/src/stores/iframe-store.ts

```diff
@@ -94,7 +94,7 @@ export default class IframeStore {
                 client.setVerificationCode('', Constants.map_action[container_key]);
             }
         } else {
-            this.iframe_url = url || '';
+            this.iframe_url = '';
```

Comment:
are we sure about this?

Reply:
```
        if (url) {
            this.iframe_url = url;

            const container = this.root_store.modules.cashier.general_store.active_container;

            if (container in Constants.map_action) {
                const container_key = container as keyof typeof Constants.map_action;

                // after we set iframe url we can clear verification code
                client.setVerificationCode('', Constants.map_action[container_key]);
            }
        } else {
            this.iframe_url = '';
        }

```

@ali-hosseini-deriv check the condition above, if `url` is a truthy value we are executing the above block, but if `url` is falsy (empty, null, undefined) then we jump to else block where `url` will always going to be `undefined` or empty